client/control/components/RecipeForm.js

```diff
@@ -57,8 +99,14 @@ export class RecipeForm extends React.Component {
       );
     }
 
+    const isCloning = route && route.isCloning;
+
+    const submitButtonCaption = recipeId && !isCloning ? 'Update Recipe' : 'Add New Recipe';
+
     return (
       <form className="recipe-form" onSubmit={handleSubmit}>
+        { this.renderCloningMessage() }
```

Comment:
This seems like an odd place to put this. It gives a really high visual priority, since it is the first thing a user will see. On the other hand, it makes sense to have it at the top of the page. Maybe the top right of the page would be better?

We could also punt on the design issue, if you'd rather do a more comprehensive redesign along with the peer approval features.